doc/installation.rst

```diff
@@ -21,19 +21,37 @@ If you are on Debian or Ubuntu Linux we recommend you try
 
 .. _install-packages:
 
+
+Using Anaconda:
+===============
+
+On all platforms, you can use Anaconda_ to install Dipy. To do so issue the following two commands in a terminal::
+
+    conda config --add channels conda-forge
+    conda install dipy
```

Comment:
What about just writing only one command?
```
conda install dipy -c conda-forge
```
And then when dipy gets linked in the main conda repo we can just remove the last two words.